buildSrc/build.gradle.kts

```diff
@@ -0,0 +1,52 @@
+import org.jetbrains.kotlin.konan.properties.loadProperties
+
+plugins {
+    `kotlin-dsl`
+    java
+}
+
+repositories {
+    maven("https://plugins.gradle.org/m2/")
+    maven("https://developer.huawei.com/repo/")
+    google()
+    mavenCentral()
+
+    maven("https://maven.pkg.github.com/Rocket-Beer/*") {
```

Comment:
why do we need this now? it doesn't have the Rocket plugin dependency which makes all of this?

Reply:
I mean for all the logic of getting properties of github from local or env, and all of this stuff. To make it easier

Reply:
I cannot use them before including the rocket-plugin, so I have to do like this, it is not beautiful. This could be better if we try to publish the plugin in the gradle repository so we shouldn't provide any credentials.